### DISCUSSION TITLE
Request for Refund Support

### DISCUSSION THREAD

**Discussion by quocduongpy** [twbs/bootstrap#40814] [Q&A]
I need support to request a refund as I mistakenly purchased the HTML theme instead of the React theme for the same subject.

This was my mistake due to my oversight, and I sincerely apologize for it.

I tried reaching out to this email address: themes@getbootstrap.com, but I have yet to receive any response.
I'm truly sorry for having to submit this information here.

If anyone reads this, please kindly review my issue. I am very grateful for your consideration.

Thank you very much.

**Comment by julien-deramond** [comment]
Hi @quocduongpy 
I converted your issue to a GH discussion here, as the themes are not maintained nor managed in this repository by the Bootstrap core team. I'm sorry for the trouble with your refund, though. Hope someone from Bootstram Themes will see this message or your emails 🤞

**Comment by quocduongpy** [comment]
It is truly unfortunate that I have yet to receive any response to my email. Everything displayed on the website, such as the refund policy and license upgrade policy, becomes meaningless when a legitimate request in accordance with these rules is ignored and disregarded.

**Reply by generalleger** [reply]
Have you gotten a response? I am trying to get a [download link](https://themes.zendesk.com/hc/en-us/articles/360001068052-Uh-oh-Where-s-my-theme-download) to a theme I purchased, and it's been longer than I thought it would take to send a link.
